cypress/integration/seededFlows/adminFlows/config/emailServerSettingsSection.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('Email Server Settings Section', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user);
+    });
+  });
+
+  describe('email server settings', () => {
+    it('updates the smtp fields', () => {
+      cy.get('@user').then(() => {
+        cy.visit('/admin/customization/config');
+        cy.findByTestId('emailServerSettings').as('emailServerSettings');
+
+        cy.get('@emailServerSettings')
```

Comment:
Since most of the action takes place within this element, you could do

```js
cy.get('@emailServerSettings').within(() => {
  // Everything in here is scoped to inside that element
  cy.findByText('Email Server Settings (SMTP)').click();
  cy.get('#settings_smtp_user_name').clear();
})
```

Reply:
✅